libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -172,6 +172,38 @@ def construct_analyzer_cmd(self, res_handler):
             LOG.error(ex)
             return []
 
+    def get_analyzer_mentioned_files(self, output):
```

Comment:
This function is called when the analyzer crashes. There is no plist output in this case.